content/spin/v2/serverless-ai-api-guide.md

```diff
@@ -50,11 +50,11 @@ The Spin SDK surfaces the Serverless AI interface to a variety of different lang
+| Operation             | Parameters                                            | Returns  | Behavior                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                   |
+| :-------------------- | :---------------------------------------------------- | :------- | :------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
```

Comment:
I know Markdown tables are horrible but this is not the answer

Reply:
Seriously I do not want to have to maintain all these dashes.  Just let the trailing column trail.